os/drivers/lcd/lcd_dev.c

```diff
@@ -65,6 +65,7 @@
 #include <stdio.h>
 #include <sched.h>
 
+#include "lcd_logo.h"
```

Comment:
Let's add this at line 73.
Priority is
1. tinyara/config.h
2. standard headers like stdio.h, unistd.h
3. tinyara specific headers like tinyara/xxx.h
4. local header like this